README.md

```diff
@@ -243,6 +246,7 @@ URI.js is published under the [MIT license](http://www.opensource.org/licenses/m
 
 ### master (will become 1.15.0)
 
+* URNs are now normalized based on the syntax given by [RFC 2141](https://www.ietf.org/rfc/rfc2141.txt)
```

Comment:
proper format and content would be: 

* fixed [`.pathname()`](http://medialize.github.io/URI.js/docs.html#accessors-pathname) to properly en/decode URN paths - ([Issue #201](https://github.com/medialize/URI.js/issues/201), [mlefoster](https://github.com/mlefoster))
* fixing URI normalization to properly handle URN paths based on  [RFC 2141](https://www.ietf.org/rfc/rfc2141.txt) syntax - ([Issue #201](https://github.com/medialize/URI.js/issues/201), [mlefoster](https://github.com/mlefoster))
  * fixed [`.normalize()`](http://medialize.github.io/URI.js/docs.html#normalize) and [`.normalizePath()`](http://medialize.github.io/URI.js/docs.html#normalize-path) to properly normalize URN paths
  * added `URI.encodeUrnPathSegment()`
  * added `URI.decodeUrnPathSegment()`
  * added `URI.decodeUrnPath()`
  * added `URI.recodeUrnPath()`